### ISSUE TITLE
[rccl] Missing gtest in makedepends

### ISSUE THREAD

**Issue by petronny** [rocm-arch/rocm-arch#478]
```
==> Starting build()...
-- The CXX compiler identification is Clang 12.0.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/rocm/hip/bin/hipcc - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR GTEST_MAIN_LIBRARY) (Required is at least version "1.10")
-- GTest not found. Downloading and building GTest.
-- Downloading/updating googletest
CMake Deprecation Warning at CMakeLists.txt:4 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


CMake Error at /usr/share/cmake-3.19/Modules/ExternalProject.cmake:2542 (message):
  error: could not find git for clone of googletest-download
Call Stack (most recent call first):
  /usr/share/cmake-3.19/Modules/ExternalProject.cmake:3430 (_ep_add_download_command)
  CMakeLists.txt:9 (ExternalProject_Add)


-- Configuring incomplete, errors occurred!
See also "/build/rccl/src/build/googletest-download/CMakeFiles/CMakeOutput.log".
CMake Error at cmake/DownloadProject.cmake:159 (message):
  CMake step for googletest failed: 1
Call Stack (most recent call first):
  cmake/Dependencies.cmake:48 (download_project)
  CMakeLists.txt:19 (include)


-- Configuring incomplete, errors occurred!
See also "/build/rccl/src/build/CMakeFiles/CMakeOutput.log".
==> ERROR: A failure occurred in build().
```

**Comment by acxz** [comment]
Seems like they made `gtest` a forced dep. https://github.com/ROCmSoftwarePlatform/rccl/commit/8c90aefb6d71793652afc3b2f26231addf7ef0dc#diff-12e8125164bbfc7556b1781a8ed516e333cc0bf058acb7197f7415be44606c72

**Comment by acxz** [comment]
Fixed with a1e1ddb
